tests/build_/test_build_request.py

```diff
@@ -1009,7 +1058,7 @@ def test_render_prod_request_v1_v2(self, registry_api_versions, scratch):
         else:
             assert get_plugin(plugins, "postbuild_plugins", "compress")
             assert get_plugin(plugins, "postbuild_plugins", "tag_from_config")
-            assert get_plugin(plugins, "exit_plugins", "koji_promote")
+            assert get_plugin(plugins, "exit_plugins",  "koji_promote")
```

Comment:
Right, there is a tool called `pylint` with much more stricter rules (including 'one space after comma') which would have caught it. That would make PRs review last much longer though :)